MekHQ/resources/mekhq/resources/Finances.properties

```diff
@@ -7,6 +7,22 @@ DistributedShares.text="%s have been distributed as shares."
 NotImplemented.text="<font color='red'><b>You cannot afford to pay %s!</b></font> Lucky for you that personnel morale is not yet implemented."
 AssetPayment.text="Your account has been credited for %s from %s "
 
+## Finances Enums
+# FinancialYearDuration Enum
+FinancialYearDuration.SEMIANNUAL.text=Semiannual
+FinancialYearDuration.SEMIANNUAL.toolTipText=The financial year lasts six months
+FinancialYearDuration.ANNUAL.text=Annual
+FinancialYearDuration.ANNUAL.toolTipText=The financial year lasts one yearl
+FinancialYearDuration.BIANNUAL.text=Biannual
+FinancialYearDuration.BIANNUAL.toolTipText=The financial year lasts two years
+FinancialYearDuration.QUINQUENNIAL.text=Quinquennial
+FinancialYearDuration.QUINQUENNIAL.toolTipText=The financial year lasts five years
+FinancialYearDuration.DECENNIAL.text=Decennial
+FinancialYearDuration.DECENNIAL.toolTipText=The financial year lasts ten years
+FinancialYearDuration.FOREVER.text=Forever
+FinancialYearDuration.FOREVER.toolTipText=The financial year lasts forever
```

Comment:
Do we really want to give this option? Some people run units from the Star League to the Succession Wars.

Reply:
I added a bit more saying it was not recommended in the name itself, but I did add that based on a direct request for it.